From 9ff6af2c397bdfbddb57567b2350c202fed45356 Mon Sep 17 00:00:00 2001
From: Felix Yan <felixonmars@archlinux.org>
Date: Mon, 31 Aug 2026 09:11:44 +0800
Subject: [PATCH] Allow uv_build 0.11.x/0.12.x (#470)

Verified that the package builds successfully with new uv_build.
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index da6d4b01..418f4570 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["uv_build>=0.9.6,<0.10.0"]
+requires = ["uv_build>=0.9.6,<0.14.0"]
 build-backend = "uv_build"
 
 [tool.uv.build-backend]
